src/core/Checkbox/Checkbox.tsx

```diff
@@ -38,17 +38,22 @@ export type CheckboxProps = {
   isLoading?: boolean;
   /**
    * Custom CSS class name for the checkmark element.
+   *
+   * @deprecated As of 1.32.0, this is now applied on the actual checkbox element.
    */
   checkmarkClassName?: string;
```

Comment:
So now there is no way to customize checkmark? At least I haven't find any usage in Azure and GitHub.

Reply:
We could mention our CSS variable but I doubt anyone wants to customize our checkmark.